src/svcutilcore/src/Resources/Strings.resx

```diff
@@ -274,55 +274,55 @@
     <value>Suppress the copyright and banner message.</value>
   </data>
   <data name="HelpDirectory" xml:space="preserve">
-    <value>Directory to create files in (default: current directory) (Short Form: /{0})</value>
+    <value>Directory to create files in (default: current directory) (Short Form: -{0})</value>
   </data>
   <data name="HelpOut" xml:space="preserve">
-    <value> The filename for the generated code. Default: derived from the WSDL definition name, WSDL service name or targetNamespace of one of the schemas. (Short Form: /{0})</value>
+    <value> The filename for the generated code. Default: derived from the WSDL definition name, WSDL service name or targetNamespace of one of the schemas. (Short Form: -{0})</value>
   </data>
   <data name="HelpReferenceCodeGeneration" xml:space="preserve">
-    <value>Reference types in the specified assembly. When generating clients, use this option to specify assemblies that might contain types representing the metadata being imported.  (Short Form: /{0})</value>
+    <value>Reference types in the specified assembly. When generating clients, use this option to specify assemblies that might contain types representing the metadata being imported.  (Short Form: -{0})</value>
   </data>
   <data name="HelpReferenceOther" xml:space="preserve">
-    <value>Add the specified assembly to the set of assemblies used for resolving type references. If you are exporting or validating a service that uses 3rd-party extensions (Behaviors, Bindings and BindingElements) registered in config use this option to locate extension assemblies that are not in the GAC.  (Short Form: /{0})</value>
+    <value>Add the specified assembly to the set of assemblies used for resolving type references. If you are exporting or validating a service that uses 3rd-party extensions (Behaviors, Bindings and BindingElements) registered in config use this option to locate extension assemblies that are not in the GAC.  (Short Form: -{0})</value>
   </data>
   <data name="HelpNostdlib" xml:space="preserve">
     <value>Do not reference standard libraries. By default mscorlib.dll and system.servicemodel.dll are referenced.</value>
   </data>
   <data name="HelpExcludeTypeCodeGeneration" xml:space="preserve">
-    <value>A fully-qualified or assembly-qualified type name to exclude from referenced contract types. (Short Form: /{0})</value>
+    <value>A fully-qualified or assembly-qualified type name to exclude from referenced contract types. (Short Form: -{0})</value>
```

Comment:
Should "-" be "--" instead? This comment is applicable to all the places where "/" are replaced with "-"